frontend/src/pages/DashboardsListPage/__tests__/DashboardListPage.test.tsx

```diff
@@ -204,4 +212,25 @@ describe('dashboard list page', () => {
 			),
 		);
 	});
+
+	it('ensure that the export JSON popover action works correctly', async () => {
+		const { getByText, getAllByTestId } = render(
+			<MemoryRouter initialEntries={['/dashbords']}>
```

Comment:
`DashboardProvider` should be added to the `render` function if required rather than wrapping your component with the same. 

any specific reason for  `MemoryRouter` wrapping ?

Reply:
Ah my bad. Both are not required in this case. Will remove them.